src/main/java/com/spotify/reaper/storage/PostgresStorage.java

```diff
@@ -488,4 +489,43 @@ public boolean updateRepairSchedule(RepairSchedule newRepairSchedule) {
       return getPostgresStorage(h).getClusterScheduleOverview(clusterName);
     }
   }
+
```

Comment:
Could it be better to implement de-centralisation as a separate capacity/concern?